### PR TITLE
Mentor guide

### PR DESCRIPTION
**Pull request by aliok**
Fixes: https://github.com/cncf/mentoring/issues/736

~~I've written the mentor guide here: https://docs.google.com/document/d/1Zozg3g5qE4jkdVEdiVtCoXvc_iuuOls1hfCgW6Pf7jQ/edit#heading=h.vpidi5cz3gv4~~

~~Happy to collect feedback there.~~

~~Later on, I will use this PR to update the markdown doc.~~

UPDATE: I've updated the doc with the content from the Google doc mentioned above

### CODE REVIEW

**Comment by aliok** [comment]
Please review @nate-double-u @jaytiaki @Riaankl @CathPag @jberkus

**Comment by Riaankl** [comment]
Add a review. Great work Ali.

**Comment by jaytiaki** [comment]
Awesome work @aliok! I've added a review too

**Comment by CathPag** [comment]
This awesome, @aliok! 💙

**Comment by aliok** [comment]
@jaytiaki @nate-double-u 

I've updated the doc with the content from the Google doc mentioned above.

Ready for final approval

File: mentors/README.md [hunk 1]
```diff
@@ -1,78 +1,140 @@
...
+  * Mentors review the proposals and provide feedback
+* Accepted mentees are [announced](https://github.com/cncf/mentoring/discussions/954)
+  * Mentees bond with the community and the mentors
+  * Mentors and mentees start working on their projects
+* TBA: (we're in the middle of the program)
+
+## Success Stories
 
-## FAQs
+TBD
```
Comment (aliok) [inline mentors/README.md:72]: Created https://github.com/cncf/mentoring/issues/1025 for this draft section. I would like to merge this PR regardless.

**Review by nate-double-u — APPROVED** [review]
Thanks for this @aliok!

**Pull request merged**